deep_regression.py

```diff
@@ -0,0 +1,251 @@
+from __future__ import absolute_import, division, print_function
+
+import tensorflow as tf
+import matplotlib.pyplot as plt
+import numpy as np
+import scipy.io as sio
+import pandas as pd
+from sklearn.utils import shuffle
+from sklearn.model_selection import train_test_split
+import tensorflow.contrib.slim as slim
+from tensorflow.contrib.framework.python.ops.variables \
+    import get_or_create_global_step
+from tensorflow.python.platform import tf_logging as logging
+
+
+'''
+Changeable parameters OR variables
+----------------------------------
+1. LEARNING_RATE
+2. reg_constant
+3. TRAINING_EPOCHS
+4. BATCH_SIZE
+5. DISPLAY_STEP
+6. test_size (splitting parameter for train:test)
+7. Hidden layer neurons number
+    N_HIDDEN_1
+    N_HIDDEN_2
+    N_HIDDEN_3
+    N_HIDDEN_4
+8. VARIANCE (for weights & biases, random normal initialization)
+9. tf.reduce_sum OR tf.reduce_mean
+10. Regularization method
+11. Gradient Descend Method
+    optimizer = tf.train.GradientDescentOptimizer()  ===> Gives all NaN values
+    optimizer = tf.train.AdamOptimizer()             ===> Gives definite values
+'''
+
+
+# =========== Parameters ============
+# LEARNING_RATE = 0.01                      # Learning rate
+BETA = 0.001                                # L2 regularization constant
+TRAINING_EPOCHS = 1000                       # EPOCHS
+BATCH_SIZE = 100                            # Set to 1 for whole sample at once
+DISPLAY_STEP = 100                          # Set to 1 for displaying all epoch
+DROPOUT_RATE = 0.9
+VARIANCE = 0.1                              # VARIANCE selection highly affects
+# Learning rate information and configuration (Up to you to experiment)
+INITIAL_LEARNING_RATE = 0.0002
+LEARNING_RATE_DECAY_FACTOR = 0.7
+NUM_OF_EPOCHS_BEFORE_DECAY = 1000
+
+# ============ Network Parameters ============
+HIDDEN_LAYERS = [500, 400, 600, 500, 500, 500]
+N_CLASSES = 1                       # Only one output prediction column
+
+
+def load_data_set(name=None):
+    """
+    This function reads the data file and extracts the features and labelled
```

Comment:
Description does not make sense. Make it clear and concise.

Reply:
I read somewhere like, in the docstring, the core job of the function should be expressed in the first line. And multiple lines description of the first line should be given on the following lines. Is that so ?